v2/kr/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
Please delete this file. We use whatever "go fmt" dictates.

Reply:
sure i'll patch.